neurons/validator_proxy.py

```diff
@@ -99,6 +101,16 @@ def authenticate_token(self, public_key_bytes):
                 status_code=401, detail="Error getting authentication token"
             )
 
+    async def healthcheck(self, request: Request):
+        authorization: str = request.headers.get("authorization")
```

Comment:
I know we discussed the auth check on all endpoints, but I'm not sure if the load balancer supports passing in a key here.

Maybe this route should be public? I think we should get @Canvinus  to weigh in.

Reply:
This is not a big issue. I can send any headers I want while health-checking using Cloudflare Load Balancer. We can keep it this way.